practice_problems_test.py

```diff
@@ -0,0 +1,12 @@
+"""Tests for practice problems."""
+
+import unittest
+
+import practice_problems
+
+class CountAppearancesTest(unittest.TestCase):
+    def test(self):
```

Comment:
could you give an example?

Reply:
sure! For example, this test could be test_good_input_results_in_expected_counts

Reply:
Done.